examples/vision/oxford_pets_image_segmentation.py

```diff
@@ -1,113 +1,218 @@
 """
-Title: Image segmentation with a U-Net-like architecture
-Author: [fchollet](https://twitter.com/fchollet)
+Title: [KerasCV] Image segmentation with a U-Net-like architecture
+Author: [fchollet](https://twitter.com/fchollet), updated by [Aritra Roy
+Gosthipaty](https://twitter.com/ariG23498) and [Margaret
+Maynard-Reid](https://twitter.com/margaretmz)
+Reviewer: [Martin Gorner](https://twitter.com/martin_gorner)
 Date created: 2019/03/20
-Last modified: 2020/04/20
+Last modified: 2023/06/19
 Description: Image segmentation model trained from scratch on the Oxford Pets dataset.
 Accelerator: GPU
 """
+
 """
-## Download the data
+This tutorial uses a U-Net like architecture for image segmentation. Data processing and
+augmentations are implemented with [KerasCV](https://keras.io/keras_cv/).
+
+U-Net was introduced in the paper, [U-Net: Convolutional Networks for Biomedical Image
+Segmentation](https://arxiv.org/abs/1505.04597/). Although U-Net is a model for image
+segmentation, it's also used in generative models such as Pix2Pix and diffusion models.
+So it's important to have a solid understanding of its architecture.
 """
 
-"""shell
-curl -O https://thor.robots.ox.ac.uk/~vgg/data/pets/images.tar.gz
-curl -O https://thor.robots.ox.ac.uk/~vgg/data/pets/annotations.tar.gz
-tar -xf images.tar.gz
-tar -xf annotations.tar.gz
 """
+## Setup and Imports
```

Comment:
please add "!pip install keras-cv" at the beginning.
I looked at existing KerasCV guides and they all do the installation differently, or don't do it at all. Let's standardize on the simplest.